docs/6.x/changelog.md

```diff
@@ -32,6 +32,13 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 6.x Releases
 
+### 6.0.4 (August 15th, 2019)
+
+#### Bugfixes
+
+* Fix an issue with ssh'ing into some nodes via Cluster Control Panel.
```

Comment:
with ssh'ing -> with ssh connections to nodes
Would be helpful to clarify `some` in this context.